### PR TITLE
VGRoid-and-Salv Additions

### PR DESCRIPTION
**Pull request by Archylle**
### **About the PR**
On the VGRoid itself two new mobs have been introduced namely the Leviathing and the Founders.
Salvage on the other hand have received new suits, the mining suits instead of the spationaut suits which were added to the salvage vend, the mining suit was reduced to be more fair for round start and the two spationaut suits were added to the vend.

With the addition of the new enemies to the VGRoid the pka received changes as well to its fire rate and to its cooldown. Additionally the maxim suit was also changed to better compensate for the new enemies introduced.

Finally the last addition is a new cloak that can be crafted in the construction menu, the goliath cloak for four goliath plates.

### **Why / Balance**
This was changed because the recent changes to the VGRoid and the goliaths have been them weaker, slower and easier to kill, this simply makes the VGRoid more contested as it means salvage will actually have to deal with threats as opposed to the slow goliaths that they can shoot from afar, additionally with the introduction of these new enemies it seemed proper to reintroduce the mining hardsuits to salvage for better protection but keeping the spationaut suits inside of the vend if a lighter armor is preferred. The Pka also received an increase to firerate and cooldown to compensate for the health of the new enemies.

The mining hardsuit piercing resistance was reduced by 20% making it 30%. The maxim suit's piercing was reduced by 20% to 50% and the blunt resistance was upped from 50 to 70%. 

The PKA's fire rate was changed from 0.5 to 0.7 and the recharge cooldown was changed from 0.75 to 0.65.

All spationaut suits in lockers were replaced with the mining hardsuits. The Goliath cloak has seen an increase in armor to 15% in blunt, piercing and slash seeing as it is from the plates of a resilient monster.

## Media
image2
image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: Added mining hardsuits to salvage roundstart
- add: Added the spationaut suits to the salvage vendor
- add: Added Leviathings and Founders to the VGRoid 
- add: Added a new craftable cloak, the goliath cloak
- remove: Removed spationaut suits from salvage round start
- tweak: Increased the PKA's fire-rate and cooldown
- tweak: maxim has been nerfed in pierce and buffed in blunt.
- tweak: mining hardsuit has been nerfed.

### CODE REVIEW

**Comment by BombasterDS2** [comment]
Round mining hardsuits affects game progression in bad way. Now there is clear progression "Early Game = Spat suit, Mid Game = Mining Suit, Late Game = Maxim suit", but you changed it to "Early Game = Mining Suit, Mid/Late Game = Maxim Suit"

Also it deprives the players of moments of joy when they find mining suit on vgroid/asteroid.

**Comment by inquisitor-star** [comment]
I did have that in mind but I did nerf the mining hardsuit so its not better than the sec suit in any way, furthermore finding the suit was either complete rng to a certain ship spawning having two or well... salv not mining and doing their jobs and just doing wrecks until they roll lucky, so one scenario the salvs fight for who gets the suits, the other the salvs don't mine and well science isn't happy. With this i nerfed it so its added against the dangers of the vgroid and its new mobs but also changed it so its not overly broken from the start.

**Review by Aidenkrz — CHANGES_REQUESTED** [review]
all of this needs goob comments

**Comment by ARMOKS** [comment]
Why not make maxim suit craftable with Goliath plates a mining suit and maybe a leviathan drop

**Comment by BasedUser** [comment]
> Round mining hardsuits affects game progression in bad way. Now there is clear progression "Early Game = Spat suit, Mid Game = Mining Suit, Late Game = Maxim suit", but you changed it to "Early Game = Mining Suit, Mid/Late Game = Maxim Suit"
> 
> Also it deprives the players of moments of joy when they find mining suit on vgroid/asteroid.

bro mining suit sucks ass

**Comment by inquisitor-star** [comment]
> all of this needs goob comments

Alright, done.

File: Resources/Prototypes/Recipes/Construction/Graphs/clothing/goliath_cloak.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- type: constructionGraph #Goobstation
```
Comment (Aidenkrz) [inline Resources/Prototypes/Recipes/Construction/Graphs/clothing/goliath_cloak.yml:1]: Shouldve mentioned before, this doesnt need a comment but it should be in goob folders.

File: Resources/Prototypes/Recipes/Construction/clothing.yml [hunk 2]
```diff
@@ -128,4 +128,15 @@
   category: construction-category-clothing
   description: Advanced security gear. Protects the station from ne'er-do-wells.
   icon: { sprite: Clothing/Head/Helmets/justice.rsi, state: icon }
+  objectType: Item
+
+- type: construction  #Goobstation
```
Comment (Aidenkrz) [inline Resources/Prototypes/Recipes/Construction/clothing.yml:133]: Shouldve mentioned before, this doesnt need a comment but it should be in goob folders.

**Review by Aidenkrz — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Recipes/Construction/clothing.yml (hunk 2 above)
Comment (inquisitor-star) [reply Resources/Prototypes/Recipes/Construction/clothing.yml]: Done as requested.

**Comment by RatherUncreative** [comment]
This is gonna make it a lot easier to recruit people into salvage. Normally finding more hardsuits isn't easy, but afaik, a salvage restock box is 1500 spesos. So, alongside everything else it's restocking, a salv vendor restock box means buying two spationaut suits for 750 spesos so that you can recruit random passengers into salvage to help you mine.

Which is nice, because you can convince passengers who might be new to the game that salvage is fun, or train people who aren't familiar with it without making them roll one of the three salvager slots. Give them a pickaxe, a hardsuit, a PKA and a gas tank, and send them to the mines. Millions of assistants must die to Goliaths.

**Review by Piras314 — APPROVED** [review]
salv is gonna kill captain with their epic hardsuits :fire: :fire:

**Comment by BombasterDS2** [comment]
Everything fine except spat suit in vend. Easy obtainable, cheap and light protected hardsuit just makes EVA suits useless. You shouldn't get a lof of department hardsuits that easily. (For example, luxury mining suit with almost same stats costs 15k spesos)

**Comment by Archylle** [comment]
> Everything fine except spat suit in vend. Easy obtainable, cheap and light protected hardsuit just makes EVA suits useless. You shouldn't get a lof of department hardsuits that easily. (For example, luxury mining suit with almost same stats costs 15k spesos)

Someone brought up a point regarding that actually, I wanted to improve the luxury mining suit but most people were against it saying its whole purpose was to show off not for protection so I do have to agree that its honestly more just to show off, I think having the spationaut suit as well as the mining suit means that salvage have the option to go lighter with some armour or heavier but be more slow.

**Comment by RatherUncreative** [comment]
> Someone brought up a point regarding that actually, I wanted to improve the luxury mining suit but most people were against it saying its whole purpose was to show off not for protection so I do have to agree that its honestly more just to show off, I think having the spationaut suit as well as the mining suit means that salvage have the option to go lighter with some armour or heavier but be more slow.

If you want to give them the option between light and heavy armor, is it possible to add both to the suit lockers? That way, they'd still have some extra suits, but you don't need to worry about the scalability issue of putting it in a vendor.

Right now, you could probably deck the whole cargo department in spationaut suits for like 7500 spesos. I don't know how viable they are for nukies, but at 750 spesos a pop, it wouldn't be hard to deck out a significant portion of the crew with spationaut suits for warops.

**Comment by Archylle** [comment]
> > Someone brought up a point regarding that actually, I wanted to improve the luxury mining suit but most people were against it saying its whole purpose was to show off not for protection so I do have to agree that its honestly more just to show off, I think having the spationaut suit as well as the mining suit means that salvage have the option to go lighter with some armour or heavier but be more slow.
> 
> If you want to give them the option between light and heavy armor, is it possible to add both to the suit lockers? That way, they'd still have some extra suits, but you don't need to worry about the scalability issue of putting it in a vendor.
> 
> Right now, you could probably deck the whole cargo department in spationaut suits for like 7500 spesos. I don't know how viable they are for nukies, but at 750 spesos a pop, it wouldn't be hard to deck out a significant portion of the crew with spationaut suits for warops.

I probably could but that would mean i need to map in new lockers for every map and add said suit to all lockers, it's a lot. I honestly think we give it a shot and see, again the spationaut hardsuit isn't as good as say body armor that you can get from the vend of the bartender, the only upside it has is that it protects from spacing.

**Comment by BombasterDS2** [comment]
> > > Someone brought up a point regarding that actually, I wanted to improve the luxury mining suit but most people were against it saying its whole purpose was to show off not for protection so I do have to agree that its honestly more just to show off, I think having the spationaut suit as well as the mining suit means that salvage have the option to go lighter with some armour or heavier but be more slow.
> > 
> > 
> > If you want to give them the option between light and heavy armor, is it possible to add both to the suit lockers? That way, they'd still have some extra suits, but you don't need to worry about the scalability issue of putting it in a vendor.
> > Right now, you could probably deck the whole cargo department in spationaut suits for like 7500 spesos. I don't know how viable they are for nukies, but at 750 spesos a pop, it wouldn't be hard to deck out a significant portion of the crew with spationaut suits for warops.
> 
> I probably could but that would mean i need to map in new lockers for every map and add said suit to all lockers, it's a lot. I honestly think we give it a shot and see, again the spationaut hardsuit isn't as good as say body armor that you can get from the vend of the bartender, the only upside it has is that it protects from spacing.

You don't need to map it on every map, just add line in locker prototype. Imo, adding mining suit round start don't fit with salvs game progression.

**Review by Piras314 — CHANGES_REQUESTED** [review]
I agree with bombaster

**Comment by Archylle** [comment]
> I agree with bombaster

Regarding the mining suit not fitting or just adding the spationaut suit into the lockers? Because I can guarantee you without the mining suits salvs will die like nothing on the vg with the new mobs?

**Comment by Piras314** [comment]
its good enough

**Pull request merged**